Resources/Public/JavaScript/bootstrap.min.js

```diff
@@ -1,7 +1,7 @@
\ No newline at end of file
+  * Bootstrap v4.1.3 (https://getbootstrap.com/)
+  * Copyright 2011-2018 The Bootstrap Authors (https://github.com/twbs/bootstrap/graphs/contributors)
+  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
```

Comment:
@albig MIT licenses are listed as "compatible" with GPL licenses: https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses.

> The MIT License is a permissive free software license originating at the Massachusetts Institute of Technology (MIT) in the late 1980s. As a permissive license, it puts only very limited restriction on reuse and has, therefore, reasonable license compatibility.

From https://en.wikipedia.org/wiki/MIT_License

Reply:
So we do not have a problem here? 

Reply:
Seems not. Also embedding MIT licensed code is common in TYPO3.